nanofi/src/core/cstream.c

```diff
@@ -153,9 +148,9 @@ int readUTFLen(uint32_t * utflen, cstream * stream) {
   return ret;
 }
 
-int readUTF(char * buf, uint64_t buflen, cstream * stream) {
+int readUTF(char * buf, uint32_t buflen, cstream * stream) {
```

Comment:
Because everywhere we call this function, we call it with an argument of type `uint32_t`, and the return value (output argument) of `readUTFLen()` is also of type `uint32_t`.